sky/utils/cli_utils/status_utils.py

```diff
@@ -491,14 +491,14 @@ def _get_replica_resources(replica_record: _ReplicaRecord) -> str:
         return '-'
     assert isinstance(handle, backends.CloudVmRayResourceHandle)
     cloud = handle.launched_resources.cloud
-    launched_resource_str = f'{cloud}'
     if handle.launched_resources.accelerators is None:
         vcpu, _ = cloud.get_vcpus_mem_from_instance_type(
             handle.launched_resources.instance_type)
-        launched_resource_str += f'(vCPU={int(vcpu)})'
+        hardware = f'vCPU={int(vcpu)})'
     else:
-        launched_resource_str += f'({handle.launched_resources.accelerators})'
-    resources_str = (f'{handle.launched_nodes}x {launched_resource_str}')
+        hardware = f'{handle.launched_resources.accelerators})'
+    spot = '[Spot]' if handle.launched_resources.use_spot else ''
+    resources_str = f'{handle.launched_nodes}x {cloud}({hardware}{spot})'
```

Comment:
```suggestion
    resources_str = f'{handle.launched_nodes}x {cloud}({spot}{hardware})'
```
nit: align with `sky.Resources.__repr__`